### PR TITLE
headless-browser: Install EventLoop and Font plugins so it doesn't crash

### LINKED ISSUE #15239: headless-browser: Crashes when trying to load websites
Discovered this when porting to Core::Stream, but it happens without my changes too.

Just `headless-browser serenityos.org` will crash it. I tried a few other urls too.

cc: @Dexesttp 

Trace:
```
175.536 headless-browser(56:56): ASSERTION FAILED: s_the
./Userland/Libraries/LibWeb/Platform/EventLoopPlugin.cpp:16
175.540 [#0 headless-browser(56:56)]: Terminating headless-browser(56) due to signal 6
175.540 [#0 Finalizer Task(5:5)]: Backtrace:
0x00000000deadc0de  Kernel::Processor::switch_context(Kernel::Thread*&, Kernel::Thread*&) + 0x383
175.540 [#0 Finalizer Task(5:5)]: Generating coredump for pid: 56
175.548 CrashDaemon(11:11): New coredump file: /tmp/coredump/headless-browser_56_1663161221
175.943 CrashReporter(57:57): Started thread "", tid = 58
177.822 [#0 VFS Sync Task(4:4)]: Ext2FS: Flushed 5 blocks to disk
181.034 CrashReporter(57:58): Generating backtrace took 5067 ms
181.034 CrashReporter(57:58): --- Backtrace for thread #0 (TID 56) ---
181.034 CrashReporter(57:58): 0x000000082c5e902b: [/usr/lib/libsystem.so] syscall2 +0xb (syscall.cpp:25 => syscall.cpp:24)
181.034 CrashReporter(57:58): 0x00000000f4f94f06: [/usr/lib/libc.so] abort +0x26 (stdlib.cpp:224)
181.034 CrashReporter(57:58): 0x00000000f4f9b20e: [/usr/lib/libc.so] __assertion_failed.localalias +0x7e (assert.cpp:33)
181.039 CrashReporter(57:58): 0x0000001ac2c826bf: [/usr/lib/libweb.so.serenity] Web::Platform::EventLoopPlugin::the() +0x1f (EventLoopPlugin.cpp:16)
181.039 CrashReporter(57:58): 0x0000001ac2c8296c: [/usr/lib/libweb.so.serenity] Web::Platform::Timer::create_repeating(int, AK::Function<void ()>&&) +0x2c (Timer.cpp:22)
181.039 CrashReporter(57:58): 0x0000001ac2b2190a: [/usr/lib/libweb.so.serenity] Web::HTML::BrowsingContext::BrowsingContext(Web::Page&, Web::HTML::BrowsingContextContainer*) [clone .localalias] +0x1fa (BrowsingContext.cpp:232)
181.043 CrashReporter(57:58): 0x0000001ac2b22b7a: [/usr/lib/libweb.so.serenity] Web::HTML::BrowsingContext::create_a_new_browsing_context(Web::Page&, JS::GCPtr<Web::DOM::Document>, JS::GCPtr<Web::DOM::Element>) +0x6a (BrowsingContext.cpp:94)
181.043 CrashReporter(57:58): 0x0000001ac2c66766: [/usr/lib/libweb.so.serenity] Web::Page::Page(Web::PageClient&) +0x56 (Page.cpp:16)
181.046 CrashReporter(57:58): 0x0000001a2672b6ef: [/bin/headless-browser] serenity_main(Main::Arguments) +0x36f (NonnullOwnPtr.h:161 => headless-browser.cpp:227 => headless-browser.cpp:53 => headless-browser.cpp:694)
181.046 CrashReporter(57:58): 0x0000001a2672a93a: [/bin/headless-browser] main +0x9a (Main.cpp:39)
181.046 CrashReporter(57:58): 0x0000001a2672ab78: [/bin/headless-browser] _entry +0x48 (crt0.cpp:43)
```

### PR DESCRIPTION
**Pull request by AtkinsSJ**
These are copied from WebContent, which isn't great, but the only place they could really go otherwise is LibWeb, which would include them in Ladybird which seems undesirable.

Fixes #15239

### CODE REVIEW

**Review by linusg — COMMENTED** [review]
Why can't these live in `LibWeb/Platform/`? Doesn't mean ladybird has to use them, but we can easily deduplicate the code 🤔

**Comment by AtkinsSJ** [comment]
> Why can't these live in `LibWeb/Platform/`? Doesn't mean ladybird has to use them, but we can easily deduplicate the code thinking

OK! I didn't require much convincing. Moved them, and used them in headless-browser.

File: Userland/Libraries/LibWeb/Platform/EventLoopPluginSerenity.h [hunk 1]
```diff
@@ -6,9 +6,9 @@
 
 #pragma once
 
-#include <LibWeb/Platform/EventLoopPlugin.h>
+#include "EventLoopPlugin.h"
```
Comment (linusg) [inline Userland/Libraries/LibWeb/Platform/EventLoopPluginSerenity.h:9]: y tho, this is not our usual style for serenity libs

**Review by linusg — COMMENTED** [review]
Much nicer!

File: Userland/Libraries/LibWeb/Platform/EventLoopPluginSerenity.h (hunk 1 above)
Comment (AtkinsSJ) [reply Userland/Libraries/LibWeb/Platform/EventLoopPluginSerenity.h:9]: Oops! I got CLion to move the files, I thought I fixed all the includes after but I obviously missed this one.

**Pull request merged**